easybuild/easyconfigs/m/MultiQC/MultiQC-0.7-goolf-1.7.20.eb

```diff
@@ -0,0 +1,78 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Bundle'
+
+name = 'MultiQC'
+version = '0.7'
+
+homepage = 'http://multiqc.info/'
+description = """Aggregate results from bioinformatics analyses across many samples into a single report"""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'optarch': False, 'pic': False}
```

Comment:
@pescobar why `'optarch': False`? If you have a specific reason for it, please include a comment in the easyconfig.

Also, the `'pic': False` is overkill, since that's the default?